comps/retrievers/opensearch/langchain/ingest.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#
+
```

Comment:
The retriever component does not need to ingest data before usage, we should let users to ingest via HTTP request. I think it's better to remove this `ingetst.py` file.